docs/blog/pydantic-v2.md

```diff
@@ -0,0 +1,221 @@
+The core logic however has remained relatively unchanged since the initial experiment.
+It's old, it smells, it needs to be rebuilt.
+The release of version 2 is an opportunity to rebuild pydantic and correct many things that don't make sense.
+
+Much of the work on V2 is already done, but there's still a lot to do.
+Now seems a good opportunity to explain what V2 is going to look like and get feedback from uses.
+
+## Headlines
+
+For good and bad, here are some of the biggest changes expected in V2.
+
+The core validation logic of pydantic V2 will be performed by a separate package 
+[pydantic-core](https://github.com/samuelcolvin/pydantic-core) which I've been building over the last few months.
+
+*pydantic-core* is written in Rust using the excellent [pyo3](https://pyo3.rs/) library which provides rust bindings
+for python.
+
+**Note:** the python interface to pydantic shouldn't change as a result of using pydantic-core, instead
+pydantic will use type annotations to build a schema for pydantic-core to use.
+
+pydantic-core is usable now, albeit with a fairly unintuitive API, if you're interested, please give it a try.
+
+pydantic-core provides validators for all common data types, 
+[see a list here](https://github.com/samuelcolvin/pydantic-core/blob/main/pydantic_core/_types.py#L291).
+Other, less commonly used data types will be supported via validator functions.
+
+### Performance :smile:
+
+As a result of the move to rust for the validation logic 
+(and significant improvements in how validation objects are structured) pydantic V2 will be significantly faster
+than pydantic V1.X.
+
+Looking at the pydantic-core [benchmarks](https://github.com/samuelcolvin/pydantic-core/tree/main/tests/benchmarks),
+pydantic V2 is between 4x and 50x faster than pydantic V1.X.
+
+In general, pydantic V2 is about 17x faster than V1.X when validating a representative model containing a range
+of common fields.
+
+### Strict Mode :smile:
+
+People have long complained about pydantic preference for coercing data instead of throwing an error.
+E.g. input to an `int` field could be `123` or the string `"123"` which would be converted to `123`.
+
+pydantic-core comes with "strict mode" built in. With this only the exact data type is allowed, e.g. passing
+`"123"` to an `int` field would result in a validation error.
+
+Strictness can be defined on a per-field basis, or whole model.
+
+#### IsInstance checks :smile:
+
+Strict mode also means it makes sense to provide an `is_instance` method on validators which effectively run
+validation then throw away the result while avoiding the (admittedly small) overhead of creating and raising
+and error or returning the validation result.
+
+### Formalised Conversion Table :smile:
+
+As well as complaints about coercion, another (legitimate) complaint was inconsistency around data conversion.
+
+In pydantic V2, the following principle will govern when data should be converted in "lax mode" (`strict=False`):
```

Comment:
If you're not going to use this terminology everywhere I'd suggest just calling it "normal" mode, or "non strict mode", if there are only two modes - normal / default, and strict.

Reply:
it's called lax throughout the code base. I'll stick with lax vs. strict.